.travis.yml

```diff
@@ -27,3 +27,4 @@ env:
     # prevent wine popup dialogs about installing additional packages
     - WINEDLLOVERRIDES="mscoree,mshtml="
     - WINEDEBUG="-all"
+    - DEBUG=electron-packager
```

Comment:
The only thing I ask is that a debug call be added that prints the options before they start getting mutated, since you're in there.